physics/solar_system_body.hpp

```diff
@@ -427,31 +457,34 @@ void SolarSystem<Frame>::RemoveOblateness(std::string const& name) {
   body->clear_reference_radius();
 }
 
+#define PRINCIPIA_SET_FIELD_FROM_OPTIONAL(field)                \
```

Comment:
I think we have something similar elsewhere, we should centralize those proto-optional-to-C++-optional macros somewhere.

Not today.